typescript/sdk/src/providers/transactions/transformer/TxTransformerTypes.ts

```diff
@@ -1,3 +1,3 @@
 export enum TxTransformerType {
-  ICA = 'Interchain Account',
+  ICA = 'interchainAccount',
```

Comment:
👍

Reply:
I'd be in favor of renaming the enum key value from `ICA` -> `InterchainAccount` for clarity

Reply:
Keen– will make update now